src/metaUtils.cxx

```diff
@@ -1010,7 +1010,7 @@ static bool MET_IsComplete(std::vector<MET_FieldRecordType *> * fields)
 //
 bool MET_Read(std::istream &fp,
               std::vector<MET_FieldRecordType *> * fields,
-              char _MET_SeperatorChar, bool oneLine, bool display_warnings,
+              char _met_SeperatorChar, bool oneLine, bool display_warnings,
```

Comment:
I personally don't like this change, or the previous change from DTI to dti, but these are minor and perhaps a matter of preference rather than standards.

Reply:
This is actually a matter of language standard, see:
https://clang.llvm.org/extra/clang-tidy/checks/bugprone-reserved-identifier.html

But I could rename to something else...